rime-simple-agent/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[project]
+name = "rime-simple-agent"
+version = "0.1.0"
+description = "Rime LiveKit Voice Agent"
```

Comment:
Not important, since it's in the name field already, but we could add the word "simple" to the description field.